tests/test_workers.py

```diff
@@ -43,9 +43,6 @@ class AcloseWorker(BaseWorker):
     """
 
     async def setup(self):
-        # TODO: call superclass setup here
-        # Make sure we haven't added this to the base class since these tests
-        # were written.
         await super().setup()
         assert not hasattr(self, "is_closed")
```

Comment:
The comment about the base class needs to stay, above the `assert not hasattr(...)` line.